Intermittently failing test for sessionTypes/MachineAuth#_getNewSessionInfo

```
  1) sessionTypes/MachineAuth
       _getNewSessionInfo
         when tokens are requested for two different audiences
           should return two different promises:

      AssertionError: expected {} to equal {}
      + expected - actual


      at Context.<anonymous> (src/sessionTypes/machineAuth.spec.js:351:33)
```

Seems to happen about every 1/20th times the tests are run, roughly. I haven't yet investigated if it is a problem with the test itself or the underlying code.

--- a/src/sessionTypes/machineAuth.spec.js
+++ b/src/sessionTypes/machineAuth.spec.js
@@ -332,7 +332,7 @@ describe('sessionTypes/MachineAuth', function() {
 
         beforeEach(function() {
           const audienceNameOne = faker.hacker.adjective();
-          const audienceNameTwo = faker.hacker.adjective();
+          const audienceNameTwo = faker.lorem.word();
           expectedPromiseOne = Promise.resolve();
           expectedPromiseTwo = Promise.resolve();
           sdk.config.audiences[audienceNameOne] = fixture.build('audience');

